clients/web/src/lib/api/queries/scrims/CurrentScrim.store.ts

```diff
@@ -3,12 +3,14 @@ import {gql} from "@urql/core";
 import {LiveQueryStore} from "../../core/LiveQueryStore";
 import {toasts} from "../../../components";
 import {screamingSnakeToHuman} from "../../../utils";
+import type {GameSkillGroup} from "@sprocketbot/core/dist/database";
```

Comment:
This should not work. Write your own interface for GameSkillGroup in this file.

Reply:
[aa5f2e2](https://github.com/SprocketBot/sprocket/pull/207/commits/aa5f2e2b64b4f390f86db78ad1fec22268c0f6d0)